uber/badge_funcs.py

```diff
@@ -31,7 +31,7 @@ def detect_duplicates():
         if session.no_email(subject):
             grouped = defaultdict(list)
             for a in session.query(Attendee).filter(Attendee.first_name != '').options(joinedload(Attendee.group)).order_by(Attendee.registered):
-                if not a.group or a.group.status != WAITLISTED:
+                if not a.group or a.group.status not in [WAITLISTED, DECLINED]:
```

Comment:
This was part of the effort to make sure nothing gets deleted, so it should be re-implemented in master.